GVRf/Framework/framework/src/main/jni/engine/renderer/renderer.cpp

```diff
@@ -408,6 +408,34 @@ RenderData* Renderer::post_effect_render_data()
     return post_effect_render_data_;
 }
 
+    RenderData* Renderer::post_effect_render_data_vulkan()
+    {
+        if (post_effect_render_data_)
+        {
+            return post_effect_render_data_;
+        }
+        float positions[18] = { -1.0f, 1.0f,  1.0f,
```

Comment:
1. Coding style: we have a magic number 18 that repeats on line 432. 
Much better way is:
declare array without explicit size (size is set by the initializer)
create separate constant positions_size = sizeof(positions) / sizeof(positions[0])
use positions_size when you need this magic number
2. Use of stack: initialized array on the stack is always suspect. Use static if you must.